packages/layout/src/tree-layout/drawing-manager/helper.js

```diff
@@ -42,7 +42,7 @@ export const removeElement = (elemID) => {
 export const resolveAligment = (context, componentData) => {
     componentData.children().forEach((component) => {
         const host = component.model().host();
-        if (host instanceof LayoutComponent && host.alignWith()) {
+        if (host instanceof LayoutComponent && host.alignWith() && context.componentMap().get(host.alignWith())) {
```

Comment:
`context.componentMap().get(host.alignWith())` has been referred multiple times. save it in a variable and use it